bytelatent/configs/entropy_model.yaml

```diff
@@ -55,7 +56,7 @@ data:
     # so pick the most efficient, so static
     patching_mode: byte
```

Comment:
cant we use this instead of a new argument? Also, there is patching_mode below, which is another thing that can be used

Reply:
They do related, but different things. PatcherArgs controls how the patching is done and can vary from entropy/bpe/space/byte/etc. `packing_mode` controls whether to pack things as byte sequences or patch sequences.